src/Microsoft.Fast.Components.FluentUI/wwwroot/DesignTokenInterop.js

```diff
@@ -0,0 +1,123 @@
+﻿import {
+    StandardLuminance,
+    bodyFont,
+    baseHeightMultiplier,
+    baseHorizontalSpacingMultiplier,
+    baseLayerLuminance,
+    controlCornerRadius,
+    density,
+    designUnit,
+    direction,
+    disabledOpacity,
+    strokeWidth,
+    focusStrokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+    typeRampMinus1FontSize,
+    typeRampMinus1LineHeight,
+    typeRampMinus2FontSize,
+    typeRampMinus2LineHeight,
+    typeRampPlus1FontSize,
+    typeRampPlus1LineHeight,
+    typeRampPlus2FontSize,
+    typeRampPlus2LineHeight,
+    typeRampPlus3FontSize,
+    typeRampPlus3LineHeight,
+    typeRampPlus4FontSize,
+    typeRampPlus4LineHeight,
+    typeRampPlus5FontSize,
+    typeRampPlus5LineHeight,
+    typeRampPlus6FontSize,
+    typeRampPlus6LineHeight,
+    accentFillRestDelta,
+    accentFillHoverDelta,
+    accentFillActiveDelta,
+    accentFillFocusDelta,
+    accentForegroundRestDelta,
+    accentForegroundHoverDelta,
+    accentForegroundActiveDelta,
+    accentForegroundFocusDelta,
+    neutralFillRestDelta,
+    neutralFillHoverDelta,
+    neutralFillActiveDelta,
+    neutralFillFocusDelta,
+    neutralFillInputRestDelta,
+    neutralFillInputHoverDelta,
+    neutralFillInputActiveDelta,
+    neutralFillInputFocusDelta,
+    neutralFillStealthRestDelta,
+    neutralFillStealthHoverDelta,
+    neutralFillStealthActiveDelta,
+    neutralFillStealthFocusDelta,
+    neutralFillStrongRestDelta,
+    neutralFillStrongHoverDelta,
+    neutralFillStrongActiveDelta,
+    neutralFillStrongFocusDelta,
+    neutralFillLayerRestDelta,
+    neutralStrokeRestDelta,
+    neutralStrokeHoverDelta,
+    neutralStrokeActiveDelta,
+    neutralStrokeFocusDelta,
+    neutralStrokeDividerRestDelta
+} from 'https://cdn.jsdelivr.net/npm/@fluentui/web-components/dist/web-components.min.js'
+
+
+export function setValueForSelector(designtoken, selector, value) {
+
+    var x = document.querySelectorAll(selector);
+    var i;
+    for (i = 0; i < x.length; i++) {
+        eval(designtoken).setValueFor(x[i], value);
```

Comment:
We need to avoid `eval` for security reasons.

Reply:
My JS knowledge lacks there. If you know how to go from string name to actual function, please enlighten me!

Reply:
@vnbaaij here's a great article on `Eval()` including the recommended replacement pattern.

Reply:
Eval is no longer being used. In fact there is no separate interop script needed anymore. Interop is now done directly with the web components script